doc/quick-reference-guide.md

```diff
@@ -0,0 +1,185 @@
+       ```
+	- For MacOS:
+		```
+		input  (iokit-name "my-keyboard-product-string")
+		output (kext)
+		```
++ `fallthrough` (boolean, defaults to `false`): allow all keys, even if not defined
+in defsrc.
+
++ `allow-cmd` (boolean, defaults to `false`): allow commands to be run as keys (very
+dangerous).
+
++ `cmp-seq` (key, defaults to `RightAlt`): compose key for Unicode input (Linux X11
+specific).
+
+	+ `cmp-seq-delay` (number): delay between each pressed key in a compose-key
+sequence
+```
+(defcfg
+  ;; For Linux
+  input  (device-file "/dev/input/by-id/usb-04d9_daskeyboard-event-kbd")
+  output (uinput-sink "My KMonad output" "/run/current-system/sw/bin/sleep 1 && /run/current-system/sw/bin/setxkbmap -option compose:ralt")
+
+  cmp-seq ralt
+  cmp-seq-delay 5
+
+  fallthrough true
+
+  allow-cmd true
+)
+```
+
++ `defsrc`: define layer of input keys
+```
+(defsrc
+  grv  1    2    3    4    5    6    7    8    9    0    -    =    bspc
+  tab  q    w    e    r    t    y    u    i    o    p    [    ]    \
+  caps a    s    d    f    g    h    j    k    l    ;    '    ret
+  lsft z    x    c    v    b    n    m    ,    .    /    rsft
+  lctl lmet lalt           spc            ralt rmet cmp  rctl
+)
+```
+
++ `defalias`: define a button, can be referenced later using `@` in `deflayer`
+```
+(defalias
+  num  (layer-toggle numbers) ;; Bind num to a button that switches to a layer
+  kil  C-A-del                ;; Bind kil to a button that Ctrl+Alt+deletes
+)
+```
+
++ `deflayer`: define as many layers as you please (to toggle, add or remove).
+```
+(deflayer qwerty
+  grv  1    2    3    4    5    6    7    8    9    0    -    =    bspc
+  tab  q    w    e    r    t    y    u    i    o    p    [    ]    \
+  caps a    s    d    f    g    h    j    k    l    ;    '    ret
+  lsft z    x    c    v    b    n    m    ,    .    /    rsft
+  lctl @num lalt           spc            ralt rmet @sym @kil
+)
+
+(deflayer numbers
+  _    _    _    _    _    _    _    _    _    _    _    _    _    _
+  _    _    _    _    _    XX   /    7    8    9    -    _    _    _
+  _    _    _    _    _    XX   *    4    5    6    +    _    _
+  _    _    \(   \)   .    XX   0    1    2    3    _    _
+  _    _    _              _              _    _    _    _
+)
+```
+
++ `layer-toggle`: toggles to a specific layer as long as the key is held.
+```
+(defalias ked (layer-delay 500 delayed-layer))
+```
+
++ `layer-delay`: change the layer temporarily
+```
+(layer-delay 500 delayed-layer)
+```
++ `layer-next`: next key press is handled by another layer (leader-key style)
+```
+(defalias ken (layer-next next-layer))
+```
++ `layer-switch`: change the base layer
+```
+(defalias kes (layer-switch switch-layer))
+```
++ `layer-add`: overlays a layer on top of the current layer  
++ `layer-rem`: removes the overlayed layer from the current layer  
+**NOTE**: ***Using them is very dangerous, it is easy to make configurations which breaks the keyboard!***
```

Comment:
This is perhaps a bit over the top, though the message is worth communicating.  I don't think we have to make it bold though :)

Reply:
Resolved in 55671fd